package.json

```diff
@@ -1,13 +1,14 @@
 {
   "name": "pillar-dao-contracts",
   "devDependencies": {
+    "@nomicfoundation/hardhat-network-helpers": "^1.0.9",
```

Comment:
Pls stick to a specific version of the package.

Reply:
done